setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
```

Comment:
Pretty sure we use line-length 79; not sure we even use flake8.  Take a look at the settings for https://github.com/lsst-sqre/mobu (in pyproject.toml)--I think those are canonical.

Reply:
Our "standard" configurations aren't in mobu — they're in the template: https://github.com/lsst/templates/tree/main/project_templates/fastapi_safir_app/example

Mobu has a few extra configs for its specific use cases.

Reply:
I pulled it out. I hope somebody will fix this because it's a nuisance to edit files when your editor is set to reformat them and there is no format configuration in the package.